docs/explanations/extension-code.md

```diff
@@ -0,0 +1,17 @@
+# Home of Plans and Devices
+
+Plans and devices can be in any pip-installable package, such as:
```

Comment:
I'd move the device/dodal part first, so that it's clear that devices belong in dodal, rather than someone adding devices to their repo then realising it's meant to be in dodal.